tests/test_web/test_api_v1.py

```diff
@@ -371,13 +358,17 @@ def test_rm_operators_overwrite_not_allowed(mock_smfsc, client, db):
     'data, error_msg',
     (
         (
-            {'from_index': 'pull:spec', 'binary_image': 'binary:image', 'add_arches': ['s390x']},
-            '"bundles" should be a non-empty array of strings',
+            {'bundles': ['some:thing'], 'from_index': 'pull:spec', 'add_arches': ['s390x']},
+            'Missing required parameter(s): binary_image',
         ),
         (
-            {'bundles': ['some:thing'], 'from_index': 'pull:spec', 'add_arches': ['s390x']},
+            {'from_index': 'pull:spec', 'add_arches': ['s390x']},
             'Missing required parameter(s): binary_image',
         ),
+        (
+            {'add_arches': ['s390x'], 'binary_image': 'binary:image'},
+            '"from_index" and "binary_image" must be specified if no bundles are specified',
```

Comment:
Let's add two more tests for when only either of them are missing.

Reply:
sorry not understanding what you want to add a test for

https://github.com/release-engineering/iib/pull/141/files#diff-abace6c6541ec4270d1bab50b005634eR369
and 
https://github.com/release-engineering/iib/pull/141/files#diff-abace6c6541ec4270d1bab50b005634eR369

Cover the cases when no bundles and:
1. from_index not set
2. binary_image not set

Reply:
(Those two links you sent refer to the same line, which is the one right above my comment)

This use case you added covers when `bundles` is not provided, but only `binary_image` is provided. Good.

We also need:

1. `bundles` not provided and only `from_index` is provided. But the kicker is that, even before these changes, if `binary_image` is not provided, then `add_arches` is required. So this test case should be: `from_index` and `add_arches` is provided and nothing else. This should get past the error "Missing required parameter(s): binary_image" and raise instead ""from_index" and "binary_image" must be specified if no bundles are specified'
1. None of `bundles`, `from_index`, `binary_image` are provided. For the same reason as the previous item, this would require the `add_arches` parameter to ensure the right error is being triggered.